scripts/utils.ts

```diff
@@ -0,0 +1,19 @@
+export const addressesDirPath = __dirname + `/../addresses`;
+
+const availableEnvironments = [
+  'testing',
+  'staging',
+  'production',
+  'hardhat',
+  '',
```

Comment:
Is there any reason, why allow the empty string?

Reply:
My initial idea was to keep `--env` optional as it was before. So the default argument of `--env` would be `""` and the output `./addresses/<CHAIN_ID>.json`.

Reply:
Makes sense to me.